index.html

```diff
@@ -22,17 +22,23 @@ <h1>Olga</h1>
           <figcaption>Olga Mishareva | Web developer</figcaption>
         </figure>
         <div>
-          <h2>Interests</h2>
+          <h3>Skills</h3>
```

Comment:
```suggestion
          <h2>Skills</h2>
```
Would change the h3 to a h2, because every headline below a h1 should be a h2.